README.md

```diff
@@ -12,13 +14,40 @@ npm install
 bower install
 ```
 
+The backend is supported by Docker.
+
+To install Docker:
+  * For Linux, go [here](https://docs.docker.com/engine/installation/). 
+  * For OS X, go [here](https://docs.docker.com/engine/installation/mac/).
+  * For Windows, go [here](https://docs.docker.com/engine/installation/windows/).
+
+You might also want to install Devtools for [React](https://facebook.github.io/react/blog/2015/09/02/new-react-developer-tools.html). For Redux, you can install the [Google Chrome extension](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd) or use the other methods described in the README [here](https://github.com/zalmoxisus/redux-devtools-extension).
+
+####To run the Docker server
+
+Go to the project directory and follow the instructions below:
+
+1. Open up the Docker Quickstart Terminal and wait for the machine to start running.
+2. On OSX, run `eval $(docker-machine env default)`.
+3. Run `docker-compose up`.
 
-You might also want to install Devtools for (React)[https://facebook.github.io/react/blog/2015/09/02/new-react-developer-tools.html] and (Redux)[https://github.com/zalmoxisus/redux-devtools-extension].
+The server should now be up and running. Initialize it by running `./init_docker.sh`. **[Perhaps not necessary to run ./init_docker.sh?]**
+
+After this, you should be able to make your first query.
+
+**Note:** If you are setting up Docker a second time, you may want to first remove the database container with `docker-compose rm db`.
```

Comment:
once my containers are seeded i find just running `docker-compose start` is all i need.